smallapp/unbound-control.c

```diff
@@ -341,6 +343,7 @@ static void print_extended(struct ub_stats_info* s)
 	PR_UL("num.query.tcpout", s->svr.qtcp_outgoing);
 	PR_UL("num.query.tls", s->svr.qtls);
 	PR_UL("num.query.tls_resume", s->svr.qtls_resume);
+	PR_UL("num.query.https", s->svr.qhttps);
```

Comment:
This PR_UL should be one line lower, after the svr.qipv6 printout, then it is in the same order as printed in the stats printout in remote.c.